### ISSUE TITLE
compatibility with Unified Code for Units of Measure

### ISSUE THREAD

**Issue by carlo-quinonez** [gentooboontoo/js-quantities#125]
I was wondering if js-quantities is compatible with the [UCUM](https://ucum.nlm.nih.gov/)? There an [online validator](https://ucum.nlm.nih.gov/ucum-lhc/demo.html) for UCOM that accepts a CSV for bulk uploads. I thought about making a CSV from the [unit definitions](https://github.com/gentooboontoo/js-quantities/blob/master/src/quantities/definitions.js) and seeing if there's any issues.

Is compatibility with a unit standard like this in scope for this library?

**Comment by gentooboontoo** [comment]
Yes, UCUM compatibility is definitely a goal for the library. I thought about it a long time ago...

It would be great to check if there are incompatibilities with the current set of units. Depending on results, it could be integrated by allowing to initialize the library with alternative sets through configuration.
